### ISSUE TITLE
Facing issue to read from file in confluence_publish_allowlist

### ISSUE THREAD

**Issue by Rakesh201180** [sphinx-contrib/confluencebuilder#835]
Hi Team,

I am facing issue to read from file when I am trying this option in conf.py to publish required documents. 

Its not reading document names from file .

confluence_publish_allowlist = 'allowed-docs.txt'

![image](https://github.com/sphinx-contrib/confluencebuilder/assets/4937405/1d3f701c-2590-401e-a2c2-e31c4d944af8)
allowed-docs.txt structure is like that

`
Test1/KB/doc1
Test1/KB/doc2
Test1/KB/doc3
Test1/KB/doc4
Test1/KB/doc5
Test1/Test1
`

but when I am putting those in below format in cong.py file its reading those file

`
confluence_publish_allowlist = ['Test1/KB/doc1',
'Test1/KB/doc2',
'Test1/KB/doc3',
'Test1/KB/doc4',
'Test1/KB/doc5',
'Test1/Test1']
` 

I tried multiple things (with hardcoded path and all) but its not taking values from file.

Need suggestions here

Thanks in advance

**Comment by jdknight** [comment]
There appears to be issues with the implementation, which may be resolved in the recent changes introduced in #838. If interested, you may try to use the [development version](https://sphinxcontrib-confluencebuilder.readthedocs.io/install/#development-installation) of this extension to see if it resolves your issue.

Alternatively, if you would like to keep using an official release, you could possibly workaround this scenario by providing an absolute path to the allow list. For example:

```
import os
doc_dir = os.path.dirname(os.path.realpath(__file__))
confluence_publish_allowlist = os.path.join(doc_dir, 'allowed-docs.txt')
```

**Comment by jdknight** [comment]
v2.3 is now available on PyPI -- marking as closed.
